### PR TITLE
docs: add CakePHP docs

### PR DESCRIPTION
**Pull request by tyler36**
## How This PR Solves The Issue

This PR adds setup guide for CakePHP projects.
I am not a CakePHP developer so I'm not sure if this the _best_ place to add the code.

Framework instructions are also sorted to make them easier find.

### CODE REVIEW

**Comment by rfay** [comment]
@ajibarra could you take a look?

**Comment by ajibarra** [comment]
@rfay @tyler36 

It looks good. To clarify there is a config/bootstrap.php file that exists in any CakePHP app that is loaded in the `parent::bootstrap()` call..not sure if profiler should be loaded before that or if it would not work. Anyway the standard bootstrap.php file included can be found here in case you want to take a look:

https://github.com/cakephp/app/blob/5.x/config/bootstrap.php

To sum up, it looks good to me :)

**Comment by tyler36** [comment]
@ajibarra Thank you for the feedback. 

Just started my CakePHP journey so I don't have the experience to tell best practises.
If I get a chance, I hope to poke a bit more.

**Comment by ajibarra** [comment]
> @ajibarra Thank you for the feedback. 
> 
> 
> 
> Just started my CakePHP journey so I don't have the experience to tell best practises.
> 
> If I get a chance, I hope to poke a bit more.
> 
> 

Well it is great to hear that!

We are hosting our annual conference next week at Luxembourg and it would be great if you would like to join us.

Of course @rfay as well and everybody in DDEV community is more than welcome to attend, so please feel free to contact us in case you want to join in person or just get a free streaming ticket.

All the info can be found here at https://cakefest.org

Don't have to say that we will be using DDEV for our workshop 😉

**Comment by tyler36** [comment]
Thank you for the invitie. Unfortunately, it's a little far from Japan. :)

Oh, you might be interest in [ddev-tinker](https://github.com/tyler36/ddev-tinker). Its a command helper for REPL. I just added CakePHP support (think "ddev cake console"). The latest killer feature is it downloads a PHP manual so you can get useful help within the console.

**Comment by ajibarra** [comment]
CakePHP community in Japan is great! We hosted the 2019 conference there at it was amazing visiting the country.

That said, I understand traveling to Luxembourg is not an option so the streaming would be the way to go :). At least the workshop day will be useful if you have just started learning the basics and even for more advanced users..it covers many topics from the framework.

About tinker, it sounds great, I will definitely take a look! Thank you!

**Comment by rfay** [comment]
Could you point me to where I can sign up for important CakePHP Announcements like the conference @ajibarra ? I can't go to many but sure would like to get to one of yours. I spoke at a Craft  CMS event last year, and often am able to get to Drupalcon.

**Comment by ajibarra** [comment]
@rfay sure!

The best source is https://cakefest.org/ usually before EOY or beginning the year the location is announced.

Additionally everything is always posted in social networks:

https://x.com/cakephp
https://www.facebook.com/CakePHP/

And we also submit the conference to php.net as well:

https://www.php.net/archive/2024.php#2024-03-22-1

The newsletter is also a good source of information, you can signup here: https://cakephp.org/

For this year's event you can get your streaming tickets here for free https://cakefest.org/tickets. I recommend the workshops sessions (Wednesday) and you can check the schedule (https://cakefest.org/schedule) for talks because you may find interesting topics there.

**Comment by rfay** [comment]
Signed up for newsletter, thanks, hope that will do it. https://cakephp.org/newsletter

**Comment by tyler36** [comment]
> The best source is https://cakefest.org/ usually before EOY or beginning the year the location is announced.

@ajibarra Thank you so much for the info. I'll be signing up too!
